src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -398,6 +398,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
                         switch (bufferType)
                         {
+                        case WINHTTP_WEB_SOCKET_BINARY_MESSAGE_BUFFER_TYPE:

```

Comment:
The bufferType they return seems to have changed to `BINARY_MESSAGE_BUFFER_TYPE`, just adding that case here _seems_ to work fine